src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -10,13 +12,30 @@
             + "Example: " + COMMAND_WORD;
 
     public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_CANCEL = "Address book is not cleared.";
```

Comment:
Good attempt in providing messages for user in various clear command scenarios. This increases user experience.